### ISSUE TITLE
Improve style guide UI

### ISSUE THREAD

**Issue by tylersticka** [cloudfour/eleventy-patterns#14]
We've discussed some improvements to the UI theme and @dromo77 did some explorations. I'd love to see this thinking continued so this can evolve from a user/reader perspective in the same way that it's evolved from a dev/maintainer perspective.

**Comment by Paul-Hebert** [comment]
That would be great! It's _very_ bare bones right now 😅

**Comment by Paul-Hebert** [comment]
@dromo77 if you'd be interested in helping out with that let me know!

**Comment by dromo77** [comment]
@Paul-Hebert Yes, this is back on my radar. If I do any more explorations I'll be sure to let you know.
